spec/require_spec.rb

```diff
@@ -1,26 +1,73 @@
 require File.dirname(__FILE__) + '/spec_helper.rb'
 require File.dirname(__FILE__) + '/require_shared.rb'
 
+shared_examples_for 'strict mode' do |method, fixture_method|
+  context 'strict error mode' do
+    context 'using global option' do
+      def with_strict_mode
+        RequireAll.strict_mode = true
+        yield
+      ensure
+        RequireAll.strict_mode = false
+      end
+
+      it "can require resolvable dependencies when there are no errors raised" do
+        with_strict_mode do
+          fixture = send(fixture_method, 'resolvable_with_strict_mode/*.rb')
```

Comment:
Do we need this glob, given that the path will eventually be passed through to `require_all` (i.e. would `... 'resolvable_with_strict_mode/')` suffice?